src/main/java/LottoGame.java

```diff
@@ -0,0 +1,50 @@
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class LottoGame {
+
+    private final Cost cost;
+    private final Lottos lottos;
+
+    public LottoGame(int cost, NumberGenerator numberGenerator) {
+        this.cost = new Cost(cost);
+        int amounts = calculateLottoAmount();
+        lottos = new Lottos(amounts, numberGenerator);
+    }
+
+    public int calculateLottoAmount() {
+        return cost.calculateLottoAmount();
+    }
+
+    // TODO: Test 코드 작성
+    public List<LottoRank> calculateResult(List<Integer> winnerNumbers, int bonusNumber) {
+        List<LottoRank> lottoRanks = new ArrayList<>();
+
+        for (Lotto lotto : lottos.getLottos()) {
+            int matchCount = lotto.getMatchCount(winnerNumbers);
+            boolean matchBonus = lotto.contains(bonusNumber);
+            lottoRanks.add(LottoRank.of(matchCount, matchBonus));
+        }
+        return lottoRanks;
+    }
```

Comment:
`lottos` 객체에게 [getter를 사용하는 대신 객체에 메시지를 보내](https://tecoble.techcourse.co.kr/post/2020-04-28-ask-instead-of-getter/)보면 어떨까요? 😃

```suggestion
    public List<LottoRank> calculateResult(List<Integer> winnerNumbers, int bonusNumber) {
        return this.lottos.calculateResult(winnerNumbers, bonusNumber);
    }
```

Reply:
추가적으로 고민해볼 부분은  `winnerNumbers`도 6자리의 로또 숫자이지 않나요?  List<Integer> 대신에 `Lotto` 타입으로 선언할 수 있지 않을까요?

Reply:
또 더 고민해볼 수 있는 부분은  

`6자리의 당첨 로또번호 + 보너스볼`가 `당첨 로또` 가 된다고 볼 수 있지 않을까요? 
이 `6자리의 당첨 로또번호`와 `보너스볼`을 인스턴스 변수로 가진 `당첨 로또` 객체를 만들면 이 객체에게 적당한 책임을 위임해줄 수 있을 것 같아요. 🙂

Reply:
getter메서드를 사용하는 대신 메시지를 보내는 방식으로 개선하였고. ```winnerNumbers```, ```bonusNumber```을 합쳐서 당첨 로또 객체를 별도로 만들었습니다. [ee7231e](https://github.com/next-step/java-lotto-kakao/pull/12/commits/ee7231ee09261e1f5953b5bcf4f6c1def2a6698f)